filter_plugins/netdata.py

```diff
@@ -19,3 +20,15 @@ def aws_creds(self, data, type_='access'):
         if type_ == 'secret':
             return config.get('default', 'aws_secret_access_key')
         return config.get('default', 'aws_access_key_id')
+
+    def redis_roles(self, services):
+        """ Organise redis services into clusters for the redis collector """
+        tmp = dict()
+        res = []
+        for service in services:
+            tmp.setdefault(service['role'], [])
+            tmp[service['role']].append(service)
+        for idx, cluster in enumerate(list(tmp.values())):
```

Comment:
`enumerate` takes an iterator, no need to use `list`
https://docs.python.org/2/library/functions.html#enumerate
`"sequence must be a sequence, an iterator, or some other object which supports iteration"`